src/content/getting-started/architecture/networkplugin-syncer/_index.en.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Network Plugin Syncer"
+---
+
+The Network Plugin Syncer provides a framework for components to  interface
+with the configured Kubernetes CNI Network plugin to perform any high level
```

Comment:
_Network_ is already part of CNI, how about:

```suggestion
with the configured Kubernetes Container Network Interface (CNI) plugin to perform any high level
```

Then we can refer to "_Kubernetes CNI_" or "CNI" elsewhere.

Reply:
Done